src/main/steps/c100-rebuild/applicant/navigationController.ts

```diff
@@ -48,6 +50,17 @@ class ApplicantNavigationController {
     let nextUrl;
 
     switch (currentPageUrl) {
+      case C100_APPLICANT_STAYING_IN_REFUGE: {
+        const applicantData = getPartyDetails(this.applicantId, this.applicantDetails) as C100Applicant;
+
+        nextUrl = applyParms(
+          applicantData.stayingInRefuge === YesOrNo.YES
```

Comment:
Add optional param for checking stayinginrefuge

Reply:
Done